tools/build.js

```diff
@@ -33,13 +33,15 @@ const transpileES6 = task('transpile-es6', () => {
   shell.exec('node node_modules/.bin/babel src --out-dir dist/ --source-maps --ignore \'**/*.spec.js\'')
 })
 
-const copyFlowSource = task('flow-copy', () => {
-  return flowCopySource(['src'], 'dist', {ignore: ['**/*.spec.js']})
+const copyFlowSource = task('flow-copy', async () => {
+  await flowCopySource(['src'], 'dist', { ignore: ['**/*.spec.js'] })
+
+  fs.copyFileSync('dist/index.js.flow', 'dist/index.umd.js.flow')
```

Comment:
Crazy. I hate that there is no uniform way to distribute flow types...

Reply:
yes. I was thinking about discarding umd modules since jest works now without it.

But I'm afraid that I missed a usage of umd modules...

Reply:
@Schedulaar do you remember one?